lightly/openapi_generated/requirements.txt

```diff
@@ -0,0 +1,5 @@
+certifi >= 14.05.14
+six >= 1.10
+python_dateutil >= 2.5.3
+setuptools >= 21.0.0
+urllib3 >= 1.15.1
```

Comment:
This introduces new requirements, they should be specified in the PIP package requirements.